docs/monitoring/service-monitoring.md

```diff
@@ -0,0 +1,16 @@
+title: Service Monitoring
+description: Sematext Service Monitoring helps you monitor the performance of all your services using one of our available integrations with automatic service discovery and monitoring capabilities.
+
+Service Monitoring is designed to help you monitor the performance and health status of your services by relying only on the Sematext Agent, without any other requirements to modify your code. Through a large number of readily available integrations, you get all the needed performance and health metrics in highly configurable dashboards or scheduled reports, specifically created for each service type. 
```

Comment:
Maybe break into a few shorter sentences?
Service Monitoring is designed to help you monitor the performance and health status of your services by relying only on the Sematext Agent, without any other requirements to modify your code. ==> Service Monitoring is for monitoring services' health status and performance metrics.  Sematext Agent automatically discovers all services that can be monitored and displays them in ____, where you can set up their monitoring without having to deploy or configure any additional agents.